src/main/kotlin/org/rust/ide/inspections/import/AutoImportFix.kt

```diff
@@ -354,8 +406,8 @@ class AutoImportFix(path: RsPath) : LocalQuickFixOnPsiElement(path), HighPriorit
         }
     }
 
-    data class Context(
-        val basePath: RsPath,
+    data class Context<Data>(
```

Comment:
I think it's better to use single letter generic name according to java conventions